app/drivers/led/is31fl3733/is31fl3733.c

```diff
@@ -0,0 +1,202 @@
+/*
+ * Copyright (c) 2020 Cameron Banna.
```

Comment:
We generally use an "AUTHOR files" approach for copyright, and would do something like:

```suggestion
 * Copyright (c) 2020 The ZMK Contributors
```

And then you can open a separate PR to add yourself to the toplevel `AUTHORS` file

Also, this would also need a license header. Is this at all based on an existing Zephyr driver you modified, or from scratch?

Reply:
Okay thanks. This is from scratch, only referenced existing drivers for how to use the api, and the datasheet for offsets.